pkg/clients/tagging/v1/client.go

```diff
@@ -69,9 +69,18 @@ func (c client) GetResources(ctx context.Context, job model.DiscoveryJob, region
 
 	if len(svc.ResourceFilters) > 0 {
 		shouldHaveDiscoveredResources = true
+
+		var tagFilters []*resourcegroupstaggingapi.TagFilter = nil
+		if len(job.SearchTags) > 0 {
+			for _, st := range job.SearchTags {
+				tagFilters = append(tagFilters, &resourcegroupstaggingapi.TagFilter{Key: &st.Key})
```

Comment:
can we add a comment here noting that the `TagFilter` without value filters only the existance of the tag in the resource? [docs](https://docs.aws.amazon.com/resourcegroupstagging/latest/APIReference/API_GetResources.html#resourcegrouptagging-GetResources-request-TagFilters)